libsks/src/pkcs11_api.c

```diff
@@ -38,16 +39,16 @@ static const CK_FUNCTION_LIST libsks_function_list = {
 	REGISTER_CK_FUNCTION(C_CloseSession),
 	REGISTER_CK_FUNCTION(C_CloseAllSessions),
 	REGISTER_CK_FUNCTION(C_GetSessionInfo),
-	DO_NOT_REGISTER_CK_FUNCTION(C_GetOperationState),
-	DO_NOT_REGISTER_CK_FUNCTION(C_SetOperationState),
+	REGISTER_CK_FUNCTION(C_GetOperationState),
```

Comment:
Ok, I will discard. It was just accidentally left in, because it is easier for me to test with client applications if all the functions are registered, because otherwise when an unimplemented function is used, the application crashes and doesn't return any information on what function was being called.